### ISSUE TITLE
Menu.group doesn't appear to work in UI schema

### ISSUE THREAD

**Issue by Stevenic** [microsoft/BotFramework-Composer#4280]
## Describe the bug

I was trying to move a custom action to its own fly-out menu vis the [UI schema](https://schemas.botframework.com/schemas/ui/v1.0/ui.schema) and I assumed that all I would have to do is add an entry like this:

```JSON
  "PagedList.dialog": {
    "form": {
      "label": "Dialog: PagedList",
      "subtitle": "Manages interactions with a paged list"
    },
    "menu": {
      "label": "Call PagedList",
      "group": "Project dialogs"
    }
  }
```

The problem is the action wouldn't show up on the fly-out at all when I set the group. I finally got it to work by setting the submenu instead:

```JSON
  "PagedList.dialog": {
    "form": {
      "label": "Dialog: PagedList",
      "subtitle": "Manages interactions with a paged list"
    },
    "menu": {
      "label": "Call PagedList",
      "submenu": ["Project dialogs"]
    }
  }
```

This got it to show up on the menu and maybe `group` just doesn't work as I expected but given that `submenu` is marked as option in the schema it seems like there's a bug here given that the action won't show up at all if you don't specify a submenu.

## Version

1.1.1

## Browser

- [x] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

**Comment by Stevenic** [comment]
I should add that I couldn't get the action to show up at all until I tried setting `"submenu": false` and then it showed up as a top level action on the fly-out menu. That's what made me realize that groups aren't actually supported yet and I had to specify a `submenu` value of some sort to get things to show up.

**Comment by yeze322** [comment]
Hi Steve, the `group` is preserved for this design. (`Bot activities`, `Dev tools`)
![image](https://user-images.githubusercontent.com/8528761/94356306-841a4600-00bf-11eb-8b66-7990fba5f8be.png)

The confusion of submenu is a problem. @a-b-r-o-w-n Andy what's your option on the naming?

**Comment by a-b-r-o-w-n** [comment]
@yeze322 I'm not sure how to address this. Do you have work scheduled for the menu groups? If not, I say we remove that from the schema for the time being.

**Comment by yeze322** [comment]
@a-b-r-o-w-n not scheduled yet, I can remove that

**Comment by Stevenic** [comment]
This seems to be resolved in 1.4 as I'm able to add actions to other flyout menus. I think this can be closed.
